db/schema.rb

```diff
@@ -251,9 +252,7 @@
     t.integer "user_id"
     t.date "last_scraped"
     t.boolean "scraper_record", default: false
-    t.text "keyword"
```

Comment:
where did this one go?

Reply:
I think that field got introduced by an early migration in the project, that was subsequently edited without being rolled back properly, so ended up in the schema forever... 

I think it would be good to have a conditional migration that drops that field if it exists (AFAIK it was never used)